src/types.d.ts

```diff
@@ -675,7 +681,17 @@ export interface ContiguousDataIndex {
 }
 
 export interface DataItemRootTxIndex {
```

Comment:
We should probably rename this to `DataItemRootIndex` since we return more than just IDs now.